alembic/versions/dc382319dd84_seed_db.py

```diff
@@ -0,0 +1,64 @@
+"""Seed DB
+
+Revision ID: dc382319dd84
+Revises: 8c089a511ab7
+Create Date: 2020-07-20 20:57:23.243476
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.sql import table, column
+from sqlalchemy import String, Integer, Date, Unicode, Boolean
+
+
+# revision identifiers, used by Alembic.
+revision = 'dc382319dd84'
+down_revision = '8c089a511ab7'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    plants_table = table('plant',
+        column('name', String),
+        column('plant_type', String),
+        column('image', Unicode),
+        column('lighting', String),
+        column('water_frequency', Integer),
+        column('harvest_time', Integer),
+        column('root_depth', Integer),
+        column('annual', Boolean)
+        )
+
+    op.bulk_insert('plants_table',
```

Comment:
Bulk insert is almost working, we are going to try to comment out all but one record to work through it. I believe we just need to figure out what `NULL` is in alembic to finish it out. We will try to get this working later today.